packages/frontend/core/src/pages/workspace/detail-page/detail-page.tsx

```diff
@@ -135,7 +137,7 @@ const DetailPageImpl = memo(function DetailPageImpl() {
   }, [doc, globalContext, isActiveView, mode]);
 
   const isInTrash = useLiveData(doc.meta$.map(meta => meta.trash));
-
+  useRegisterFindInPageCommands();
```

Comment:
I think CMD+F should work universally that searches ALL texts, including texts in all routes.